src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -98,6 +98,18 @@ def delete_fs_volume(self, volname, confirm):
                 "that is what you want, re-issue the command followed by " \
                 "--yes-i-really-mean-it.".format(volname)
 
+        ret, out, err = self.mgr.check_mon_command({
+            'prefix': 'config get',
+            'key': 'mon_allow_pool_delete',
+            'who': 'mon',
+            'format': 'json',
+        })
+        mon_allow_pool_delete = json.loads(out)
+        if not mon_allow_pool_delete:
+            return -errno.EPERM, "", "pool deletion is disabled; you must first " \
+                "set the mon_allow_pool_delete config option to true before you " \
+                "can destroy a pool"
```

Comment:
`s/you can destroy a pool/volumes can be deleted/`

Reply:
Done